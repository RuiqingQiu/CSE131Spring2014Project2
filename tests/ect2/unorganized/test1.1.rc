int a = 10;
float f = 10.5;
bool b = true;
bool c = false;

function : int foo(){
   decltype(a) k;
   return 10;
}

function : void main(){
   cout << "a: " <<  a  << " f: " << f << "\n" << b << " " << c << " test" << endl;
   cout << true << false << endl;
}