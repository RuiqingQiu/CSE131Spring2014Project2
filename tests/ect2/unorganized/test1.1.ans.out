a: 10 f: 10.50
true false test
truefalse
